mobile-widgets/qmlmanager.cpp

```diff
@@ -1436,14 +1434,9 @@ QStringList QMLManager::buddyInit() const
 	int i = 0;
 	for_each_dive (i, d) {
 		QString temp = d->buddy;
-		if (!temp.isEmpty() && !temp.contains(",")){
+		if (!temp.isEmpty()){
 			buddies << d->buddy;
 		}
```

Comment:
This is the initial list creation when reading the divelog, if there is a dive in the divelog that has more than one buddy it is already a list. In which case it just adds to the big list of buddies.